features/charlie_creates_a_petition.feature

```diff
@@ -201,3 +201,20 @@ Scenario: Charlie creates a petition when blocked
   When I press "Yes – this is my email address"
   Then a petition should not exist with action: "The wombats of wimbledon rock.", state: "pending"
   And a signature should not exist with email: "womboid@wimbledon.com", state: "pending"
+
+Scenario: Charlie creates a petition when his IP address is rate limited
+  Given the burst rate limit is 1 per minute
+  And there are no allowed IPs
+  And there are no blocked IPs
+  And there are 2 petitions created from this IP address
```

Comment:
two petitions, not one?

Reply:
Yes, because the creator signature hasn't been created yet so will pass the `<` check if it's only one.